src/main/scala/com/amazon/deequ/analyzers/MinLength.scala

```diff
@@ -41,4 +48,22 @@ case class MinLength(column: String, where: Option[String] = None)
   }
 
   override def filterCondition: Option[String] = where
+
+  @VisibleForTesting
```

Comment:
Is this needed if we are using package private modifier and have the test in the appropriate package?